ikasaneip/security/ldap/src/main/java/org/ikasan/security/service/LdapServiceImpl.java

```diff
@@ -78,6 +79,7 @@
 public class LdapServiceImpl implements LdapService
 {
 	private static Logger logger = LoggerFactory.getLogger(LdapServiceImpl.class);
+    private static final CharsetEncoder VALID_CHARSET_ENCODER = Charset.forName(CharEncoding.ISO_8859_1).newEncoder();
```

Comment:
@veretie why is ISO_8859_1 a VALID_CHARSET_ENCODER why  should we use CharEncodeing.UTF-8 ?

Reply:
IKASAN already uses ISO_8859_1 charset in current source version, for user's account name check - I think this could have been intended to prevent encoding issues with Sybase. Using IKASAN v2 + Sybase I came across same issue on different ldap user fields, so I have extended the check for department and roles.